modules/perception/obstacle/fusion/probabilistic_fusion/pbf_base_track_object_matcher.h

```diff
@@ -84,5 +84,4 @@ class PbfBaseTrackObjectMatcher {
 }  // namespace perception
 }  // namespace apollo
 
-#endif  // MODULES_PERCEPTION_OBSTACLE_FUSION_PROBABILISTIC_FUSION_PBF_BASE_TRACK_OBJECT_MATCHER_H_
-        // // NOLINT
+#endif  // MODULES_PERCEPTION_OBSTACLE_FUSION_PROBABILISTIC_FUSION_PBF_BASE_TRACK_OBJECT_MATCHER_H_ // NOLINT
```

Comment:
maybe you can put `clang-format off` and `clang-format on` like the following
// clang-format off
+#endif  // MODULES_PERCEPTION_OBSTACLE_FUSION_PROBABILISTIC_FUSION_PBF_BASE_TRACK_OBJECT_MATCHER_H_ // NOLINT
// clang-format on